migrations/5_oraclize.js

```diff
@@ -0,0 +1,9 @@
+const OraclizeWrapper = artifacts.require('OraclizeWrapper');
```

Comment:
I'm not exactly sure about the migration naming convention. Is `<number>_<contract name>` sufficient? Does it assume by default that we're deploying the contract? I'd rather see e.g. `5_deploy_oraclize.js`. I know it's a minor remark but I don't think that the current convention will scale very well over time.